_meta/beat.reference.yml

```diff
@@ -20,3 +20,10 @@ apm-server:
   #ssl.enabled: false
   #ssl.certificate : "path/to/cert"
   #ssl.key : "path/to/private_key"
+
+  # Comma separated list of permitted origins for frontend. User-agents will send
+  # a origin header that will be validated against this list.
+  # An origin is made of a protocol scheme, host and port, without the url path.
+  # Allowed origins in this setting can have * to match anything (eg.: http://*.example.com)
```

Comment:
Nice!

Would `http://*.example.com` actually work? Haven't seen it before and can't find that in the spec. Seems to allow only either the exact domain or the wildcard `*`, but not the mix. Maybe I missed something?

Reply:
not yet, that's work for another PR...

> 2. Wildcard pattern matching
> I can do 2 in a separated PR, but I wouldn't go with filepath.Match...
> OK let's push (2) to a separate PR

Reply:
OK. Let's add this line when it works. If you remove it for now, I'll merge 💪

Reply:
done....